frontend/src/components/podcast/episode-table/episode-table.tsx

```diff
@@ -0,0 +1,40 @@
+import { Table } from 'components/common/common';
```

Comment:
move it to `frontend/src/components/podcast/compnents/episode-table/episode-table.tsx`